src/Hl7.Fhir.Specification.Tests/Snapshot/SnapshotGeneratorTest.cs

```diff
@@ -8267,5 +8267,138 @@ private void logExtensions(string title, IEnumerable<Extension> extensions, int
                     Debug.WriteLine($"{new string(' ', level * 3)}{extension.Url} : {extension.Value}");
             }
         }
+
+        [TestMethod]
+        public async T.Task TestMergingAPreviouslyRemovedElement()
+        {
+            var baseStructureDefinition = new StructureDefinition()
```

Comment:
To prevent large unit test files, it would be better to add these profiles to the folder TestData and use them here. You can use then the already existing `_testResolver` as well.